tgui/packages/tgui/interfaces/AtmosControl.js

```diff
@@ -125,7 +125,7 @@ const AtmosControlMapView = (_properties, context) => {
   return (
     <Box height="526px" mb="0.5rem" overflow="hidden">
       <NanoMap onZoom={v => setZoom(v)}>
-        {alarms.filter(a => a.z === 1).map(aa => (
+        {alarms.filter(a => a.z === 2).map(aa => (
```

Comment:
Do we have to hardcode the z level here? It might be better to just get the station z level directly.

Reply:
This is TGUI so you either pass forward redundant data or hardcode it.

Normal DM procs dont work in TGUI

Reply:
It will be a little annoying, but yeah I could pass in `level_name_to_num(MAIN_STATION)` from DM if its wanted.

Reply:
Given how infrequently these are changed, I am deeming this not worth doing